src/DynamicMvvm.Uno.WinUI/DynamicMvvm.Uno.WinUI.csproj

```diff
@@ -0,0 +1,54 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
+	<PropertyGroup>
+		<TargetFrameworks>net6.0-windows10.0.18362;netstandard2.0;net6.0-android;net6.0-ios;</TargetFrameworks>
+		<LangVersion>10.0</LangVersion>
+		<!-- Ensures the .xr.xml files are generated in a proper layout folder -->
+		<GenerateLibraryLayout>true</GenerateLibraryLayout>
+		<RootNamespace>Chinook.DynamicMvvm</RootNamespace>
+		<Authors>nventive</Authors>
+		<Company>nventive</Company>
+		<AssemblyName>Chinook.DynamicMvvm.Uno.WinUI</AssemblyName>
+		<PackageId>Chinook.DynamicMvvm.Uno.WinUI</PackageId>
+		<Description>Chinook.DynamicMvvm is a collection of extensible MVVM libraries for declarative ViewModels.</Description>
+		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<PackageReadmeFile>README.md</PackageReadmeFile>
+		<PackageTags>mvvm;ios;android;chinook;net-6.0;maui;winui;</PackageTags>
```

Comment:
I think we can remove net-6.0. All nuget packages are for .net and the platforms are already interpreted by nuget.org